homeassistant/components/onvif/coordinator.py

```diff
@@ -0,0 +1,43 @@
+"""Coordinator for the ONVIF integration."""
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from zeep.helpers import serialize_object
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+from .device import ONVIFDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+type OnvifConfigEntry = ConfigEntry[OnvifDataUpdateCoordinator]
+
+
+class OnvifDataUpdateCoordinator(DataUpdateCoordinator[dict[Any, Any]]):
```

Comment:
The coordinator only handles fetching imaging settings, I'd suggest to make that clear in the class name.